Explorer/Assets/DCL/Character/CharacterCamera/Components/CameraComponent.cs

```diff
@@ -8,24 +8,23 @@ namespace DCL.CharacterCamera
     /// </summary>
     public struct CameraComponent
     {
+        private CameraMode mode;
         public CameraMode Mode
         {
-            get => currentMode;
+            get => mode;
             set
             {
-                if (value == currentMode) return;
+                if (value == mode) return;
 
-                PreviousMode = currentMode;
-                currentMode = value;
+                PreviousMode = mode;
+                mode = value;
```

Comment:
Changing a file for nothing. Boo.

Reply:
haha yes I just wanted to "correct" the naming of the variable I created in a previous PR